src/styles/main.css

```diff
@@ -1,57 +1,57 @@
 @font-face {
-  font-family: 'Gotham Rounded';
+  font-family: "Gotham Rounded";
```

Comment:
@victorgdev Why did you decide to change single quotes to double quotes? I'm asking because @lucca65 did [the opposite](https://github.com/cambiatus/frontend/commit/d9391e4a7481419df23b6c35819e7bd2a2a6088b#diff-75ab587240a5ab47430118a1d96059c3) some time ago :)

Reply:
Hi @amiskov I didn't do it on purpose, when I had the yarn install/start done these changes appeared and I was unsure if I should revert or not. Should I revert it?